### DISCUSSION TITLE
Adding Image and columns in Rich Text Editor

### DISCUSSION THREAD

**Discussion by KeerthanaAnnamalai** [mantinedev/mantine#6131] [Feature requests]
Just like other rich text editor like **Quill** or **Tiptap** or someother, if we have image addition will be better. Along with that, if we have a Column option like wordpress, it will become a powerful Rich text editor

**Comment by rtivital** [comment]
Mantine rich text editor component is based on tiptap, you can use any tiptap extensions with it
